packages/clerk-js/src/ui/components/OrganizationSwitcher/OrganizationSwitcherTrigger.tsx

```diff
@@ -27,12 +27,13 @@ export const OrganizationSwitcherTrigger = React.forwardRef<HTMLButtonElement, O
             gap={3}
             size={'sm'}
             organization={organization}
-            sx={{ maxWidth: '30ch' }}
+            sx={{ maxWidth: '30ch', minWidth: '190px' }}
```

Comment:
Can you elaborate? This midwidth here looks unrelate to the billing feature

Reply:
@nikosdouvlis we need a min-width in case we have an org name with one letter.

Reply:
I don't really understand. What occurs when there's an org with one letter?

Reply:
@nikosdouvlis @desiprisg this was a requirement from @SokratisVidros but since it interferes with the public lib I guess we should ditch it. Plus the breadcrumbs in the dashboard don't have a fixed width either.

Reply:
@raptisj The fixed width of the <OrganizationSwitcher/> should be as styling applied from Dashboard, not added to the component for everyone. The component must be as flexible as possible. We have the appearance prop for that so that we can customize the styling when we use it.